packages/transfer-app/src/Transfer.tsx

```diff
@@ -4,7 +4,7 @@
 
 import { Container, Header } from '@substrate/ui-components';
 import React from 'react';
-import { Route, RouteComponentProps, Switch } from 'react-router-dom';
+import { Route, RouteComponentProps, Switch, Redirect } from 'react-router-dom';
```

Comment:
nit: Alphabetical ordering of imports `import { Redirect, Route, RouteComponentProps, Switch } from `react-router-dom`